source/Octopus.Server.Client/Model/BuildInformation/OctopusPackageVersionBuildInformationMappedResource.cs

```diff
@@ -19,7 +19,7 @@ public class OctopusPackageVersionBuildInformationMappedResource : Resource
 
         public string IssueTrackerName { get; set; }
         public WorkItemLink[] WorkItems { get; set; }
-        public CommitDetail[] Commits { get; set; }
+        public CommitDetails[] Commits { get; set; }
```

Comment:
@slewis74 Do these require a major bump?

Reply:
yeah, will be a major bump